helix-term/src/ui/mod.rs

```diff
@@ -331,7 +331,12 @@ pub mod completers {
                         path.push("");
                     }
 
-                    let path = path.to_str().unwrap().to_owned();
+                    let path = if cfg!(windows) && is_dir {
+                        // Convert Windows style path separator to Unix style
+                        path.to_str().unwrap().replace("\\", "/")
```

Comment:
Why only do it if it is a directory?

Reply:
Because only directories end with a path separator
for example:
```
    file:       file.ext
    directory:  directory\
```